edition/voxel_tool_terrain.cpp

```diff
@@ -235,6 +236,14 @@ void VoxelToolTerrain::_set_voxel_f(Vector3i pos, float v) {
 	_terrain->get_storage().set_voxel_f(v, pos, _channel);
 }
 
+void VoxelToolTerrain::set_strength(float strength) {
+	_strength = math::clamp(strength, 0.001f, 1.f);
```

Comment:
This is just a style nitpick, but I'd prefer using both leading and trailing zeros for float literals.
```suggestion
	_strength = math::clamp(strength, 0.001f, 1.0f);
```